client/src/pages/Home.js

```diff
@@ -2,6 +2,7 @@ import "./Home.css";
 import User from "../component/User";
 import MilestoneTable from  "../component/MilestoneTable";
 
+
```

Comment:
No need to add a line here. Its worth checking the PR yourself first to make sure there is not anything in there you didnt intend